ci/travis_install_conda.sh

```diff
@@ -19,28 +19,31 @@
 
 set -e
 
-if [ $TRAVIS_OS_NAME == "linux" ]; then
-  MINICONDA_URL="https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh"
-else
-  MINICONDA_URL="https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh"
-fi
+if (! which conda > /dev/null ); then
```

Comment:
This checks for an existing conda but this is never the case on CI, isn't it?

Reply:
This script can now be invoked several times per job and we don't want to install miniconda more than once.